feature/settings/src/main/java/com/google/jetpackcamera/settings/SettingsViewModel.kt

```diff
@@ -362,6 +363,22 @@ class SettingsViewModel @Inject constructor(
             )
         }
 
+        if (currentSettings.videoQuality != VideoQuality.AUTO &&
+            newLensConstraints.supportedVideoQualitiesMap[DynamicRange.SDR]?.contains(
+                currentSettings.videoQuality
+            ) != true
+        ) {
+            return FlipLensUiState.Disabled(
+                currentLensFacing = currentSettings.cameraLensFacing,
+                disabledRationale = StabilizationUnsupportedRationale(
```

Comment:
I don't think `StabilizationUnsupportedRationale` is the right rationale to use here. You'll need to create a new one for video quality.